src/main/scala/com/cognite/spark/datasource/RawTableRelation.scala

```diff
@@ -57,7 +57,10 @@ class RawTableRelation(
 
   override val schema: StructType = userSchema.getOrElse {
     if (inferSchema) {
-      val rdd = readRows(inferSchemaLimit, collectSchemaInferenceMetrics)
+      val rdd =
+        readRows(
+          inferSchemaLimit.orElse(Some(Constants.DefaultRawBatchSize)),
```

Comment:
Better make a new constant for this (but this value, 10k, seems fine).

Reply:
Maybe, I thought batch size always would be a good metric for infer schema limit, maybe even use the user provided batch size for this. What do you think about that?